release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -694,6 +694,26 @@ In this release, the Node Tuning Operator introduces support for deferring tunin
 
 For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#defer-application-of-tuning-changes_node-tuning-operator[Deferring application of tuning changes].
 
+[id="ocp-release-notes-scalability-and-performance-nrop-policy_{context}"]
+==== NUMA Resources Operator now uses default SELinux policy
+
+With this release, the NUMA Resources Operator no longer creates a custom SELinux policy to enable the installation of Operator components on a target node. Instead, the Operator uses a built-in container SELinux policy. This change removes the additional node reboot that was previously required when applying a custom SELinux policy during an installation. 
+
+[IMPORTANT]
+====
+During an upgrade to {product-title} 4.18, the NUMA Resources Operator removes the `MachineConfig` resource that previously applied the custom SELinux policy. This action triggers one additional reboot for each configured node. Also, resource topology exporter pods might temporarily enter a `CrashLoopBackOff` state during this process. This is expected behavior as the Operator transitions to the built-in SELinux policy. If you need to defer the additional reboot you can apply an annotation before upgrading to {product-title} 4.18. Use the following patch command to apply the annotation:
+
+[source,terminal]
+----
+$ oc patch numaresourcesoperators.nodetopology.openshift.io numaresourcesoperator \   
+  --type='json' -p '[{"op": "add", "path": "/spec/nodeGroups/0/annotations", "value": {"config.node.openshift-kni.io/selinux-policy":"custom"}}]'
+----
+
+This patch example will patch the `numaresourcesoperators` object named `numaresourcesoperator` adding the `config.node.openshift-kni.io/selinux-policy:"custom"` annotation to the `nodeGroups` object at index `0`.
```

Comment:
```suggestion
This patch example will patch the `numaresourcesoperators` object named `numaresourcesoperator` adding the `config.node.openshift-kni.io/selinux-policy:"custom"` annotation to the `nodeGroups` object at an index of `0`.
```

Reply:
I wasn't sure if we needed this for clarity.

Reply:
think this is ok